src/components/explorer/IndividualMeasurements.js

```diff
@@ -0,0 +1,232 @@
+import React, { useCallback, useMemo } from "react";
+import PropTypes from "prop-types";
+import { individualPropTypesShape, measurementPropTypesShape } from "../../propTypes";
+import { ontologyTermSorter, useIndividualPhenopacketDataIndex } from "./utils";
+import { RoutedIndividualContent, RoutedIndividualContentTable } from "./RoutedIndividualContent";
+import { EM_DASH } from "../../constants";
+import { Descriptions, Table } from "antd";
+import OntologyTerm, { conditionalOntologyRender } from "./OntologyTerm";
+import { Procedure } from "./IndividualMedicalActions";
+import TimeElement from "./TimeElement";
+
+
+const FLEX_COLUMN_STYLE = {
+    "display": "flex",
+    "flexDirection": "column",
```

Comment:
add a gap to this flex colum to make sure columns are spaced out nicely - could use `1em` for the width of an m-character 

`"gap": "1em",`